framework/src/main/java/org/checkerframework/common/aliasing/jdk.astub

```diff
@@ -0,0 +1,7 @@
+import org.checkerframework.common.aliasing.qual.*;
+
+package java.lang;
+
+class Object {
+    @Unique Object();
```

Comment:
Why is this annotation being added in a `jdk.astub` file rather than in the annotated JDK, https://github.com/typetools/jdk?

Reply:
You should link related PRs from the PR descriptions, so that it is immediately clear that PRs need to be merged together.

You should remove this jdk.astub file from this PR now, as the changes from the related PR should be picked-up automatically.

Make the one suggested update in the jdk project and let's see whether the build just needs to re-run.

Reply:
I have removed the jdk.astub file from this PR and provided the link for the PR in description which needs to be merged together.